netlify.toml

```diff
@@ -18,4 +18,4 @@
   # Default build command.
   command = "npm ci; cp .env-dist .env; npm run build-storybook"
 
-  environment = { NODE_VERSION = "18.12.1", NPM_VERSION = "9.6.5" }
+  environment = { NODE_VERSION = "20.10.0", NPM_VERSION = "10.2.4" }
```

Comment:
❤️ This made me actually write a wrapper around https://nodejs.org/download/release/index.json to see what the latest versions were, their matching node+npm versions, release dates, and whether it's an LTS and/or security release.

Looks like Node v20.10.0 includes npm 10.2.3. Why a sliightly different npm version here? Although I guess it does match our slightly mismatched Volta config. 🤔 

```sh
npx pdehaan/node-versions-md 20 | head -8
```

version  | npm      | date       | lts        | security
--------:|---------:|------------|------------|------------
v20.10.0 | 10.2.3   | 11/21/2023 | Iron       |
v20.9.0  | 10.1.0   | 10/23/2023 | Iron       |
v20.8.1  | 10.1.0   | 10/12/2023 |            | (SECURITY)
v20.8.0  | 10.1.0   | 9/27/2023  |            |
v20.7.0  | 10.1.0   | 9/17/2023  |            |
v20.6.1  | 9.8.1    | 9/6/2023   |            |

Looks like current package.json in `main` has:
https://github.com/mozilla/blurts-server/blob/26ed0928de671eac749ed27ff1c6fc3cc3c88177/package.json#L5-L8
https://github.com/mozilla/blurts-server/blob/26ed0928de671eac749ed27ff1c6fc3cc3c88177/package.json#L46-L49